lib/npg_tracking/data/reference/find.pm

```diff
@@ -364,22 +365,33 @@ sub lims2ref {
 =head2 parse_reference_genome
 
 Parses LIMs notation for reference genome, returns a list containing
-an organism, strain (genome version) and, optionally,
-a transcriptome version or an empty list.
+an organism, strain (genome version) and, optionally, a transcriptome
+version and/or an aligner; or an empty list.
 
 =cut
 sub parse_reference_genome {
-  my ($self, $reference_genome) = @_;
-  $reference_genome ||= $self->reference_genome;
-  if ($reference_genome) {
-     ##also allows for transcriptome version e.g. 'Homo_sapiens (1000Genomes_hs37d5 + ensembl_release_75)'
-     my @a = $reference_genome  =~/  (\S+) \s+ [(]  (\S+) (?: \s+ \+ \s+ (\S+) )? [)]  /smx;
-     if (scalar @a >= 2 && $a[0] && $a[1]) {
-        if (! $a[2]) { $#a = 1; }
-        return @a;
-     }
-  }
-  return;
+    my ($self, $reference_genome) = @_;
+    $reference_genome ||= $self->reference_genome;
+    if ($reference_genome) {
+        ##also allows for transcriptome version and aligner e.g. 'Homo_sapiens (1000Genomes_hs37d5 + ensembl_release_75) [star]'
+        ## no critic (Variables::ProhibitPunctuationVars RegularExpressions::ProhibitComplexRegexes)
```

Comment:
`$LAST_PAREN_MATCH{organism}` rather than `$+{organism}` will avoid triggering `Variables::ProhibitPunctuationVars` - though it is admitedly a little more verbose....